code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -88,6 +88,9 @@
 	burst_size = 2
 	pin = /obj/item/device/firing_pin/implant/pindicate
 
+/obj/item/weapon/gun/projectile/automatic/c20r/unrestricted
+	pin = /obj/item/device/firing_pin

```

Comment:
oh w/e

you can totally have objects in a definition btw.
it's done in tons of places.